notebooks/official/experiments/get_started_with_custom_training_autologging_local_script.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    if delete_bucket or os.getenv("IS_TESTING"):
</pre><p>I think you meant delete_experiment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1887/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>done</p>